src/main/java/com/kodilla/ecommercee/service/ProductService.java

```diff
@@ -12,15 +12,15 @@
 @Service
 @Transactional
 public class ProductService {
-    private ProductRepository productRepository;
+    private final ProductRepository productRepository;
 
     @Autowired
-    public ProductService(ProductRepository productRepository) {
+    public ProductService(final ProductRepository productRepository) {
```

Comment:
https://stackoverflow.com/questions/2236599/final-keyword-in-method-parameters

Reply:
no to przepraszam za tego babola

Reply:
to nie babol :) po prostu to nic nie wnosi, może być lub nie ;)

Reply:
a im mniej kodu tym lepiej, bo nie ma miejsca na pomyłkę